code/__DEFINES/components.dm

```diff
@@ -57,6 +57,7 @@
 #define COMSIG_ATOM_ATTACK_GHOST "atom_attack_ghost"			//from base of atom/attack_ghost(): (mob/dead/observer/ghost)
 	#define COMPONENT_NO_ATTACK_HAND 1							//works on all attack_hands.
 #define COMSIG_PARENT_EXAMINE "atom_examine"                    //from base of atom/examine(): (/mob)
+#define COMSIG_ATOM_EX_ACT "atom_ex_act"						//from base of atom/ex_act(): (severity, target)
```

Comment:
Not sure what this signal is for

Reply:
Nothing uses it yet, but the source is there now, /tg/ does it via the base atom proc, just like the attack_hands and such